prog/vm/vm_pool.rb

```diff
@@ -56,6 +56,7 @@ def before_run
 
   label def destroy
     vm_pool.vms.each do |vm|
+      vm.private_subnets.each { _1.incr_destroy }
```

Comment:
So, while this may be expedient....is this traversal concurrency-safe? Can `vm.private_subnets` mutate after this point? We tend to be doing this more than once, all of these will come back to roost, e.g. https://github.com/ubicloud/ubicloud/pull/613#issuecomment-1721377006

Reply:
Since these are from our internal VM pool project and we've created a distinct private subnet for each VM, I don't anticipate changes in these private subnets moving forward. However, this holds true for customer projects.